js/aside-script.js

```diff
@@ -211,6 +211,22 @@ function UpdateLocale()
 	);
 }
 
+chrome.storage.onChanged.addListener((changes, namespace) =>
+{
+	if (namespace == "sync")
+		for (key in changes)
+			if (key === "sets")
+				chrome.runtime.sendMessage({ command: "loadData" }, (collections) =>
+				{
+					document.querySelector(".tabsAside section h2").removeAttribute("hidden");
+					document.querySelectorAll(".tabsAside section > div").forEach(i => i.remove());
+
+					if (document.querySelector(".tabsAside.pane section > div") == null)
+						collections.forEach(i =>
```

Comment:
Collections can be returned undefined from background js, crashing this .forEach